@xen-orchestra/babel-config/index.js

```diff
@@ -42,8 +42,8 @@ const getConfig = (key, ...args) => {
   return config === undefined
     ? {}
     : typeof config === 'function'
-    ? config(...args)
-    : config
+      ? config(...args)
+      : config
```

Comment:
Please make sure your deps are up to date and recreate a commit to reformat the code with the new Prettier.

I really don't understand why this untouched file appears in your PR, maybe a bad merge/restart?

Reply:
I suspect that husky did something during the merge.